src/components/hint/index.tsx

```diff
@@ -496,11 +496,7 @@ class Hint extends Component<HintProps, HintState> {
   }
 
   render() {
-    const {visible, onBackgroundPress, testID} = this.props;
-
-    if (!visible) {
-      return this.props.children;
-    }
```

Comment:
This is intention and not a bug (: 
When the Hint is not visible we want to return the original item as it is.